<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List
href="comparing%20structure%20predictions%20log_files/filelist.xml">
<link rel=Edit-Time-Data
href="comparing%20structure%20predictions%20log_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Nanda Lab</o:Author>
  <o:Template>Normal</o:Template>
  <o:LastAuthor>Nanda Lab</o:LastAuthor>
  <o:Revision>44</o:Revision>
  <o:TotalTime>533</o:TotalTime>
  <o:Created>2012-04-06T19:01:00Z</o:Created>
  <o:LastSaved>2012-04-14T00:47:00Z</o:LastSaved>
  <o:Pages>5</o:Pages>
  <o:Words>3981</o:Words>
  <o:Characters>22694</o:Characters>
  <o:Company>Microsoft</o:Company>
  <o:Lines>189</o:Lines>
  <o:Paragraphs>53</o:Paragraphs>
  <o:CharactersWithSpaces>26622</o:CharactersWithSpaces>
  <o:Version>14.00</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData
href="comparing%20structure%20predictions%20log_files/themedata.thmx">
<link rel=colorSchemeMapping
href="comparing%20structure%20predictions%20log_files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
  </w:Compatibility>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="267">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1107305727 0 0 415 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-520092929 1073786111 9 0 415 0;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-520081665 -1073717157 41 0 66047 0;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-1593833729 1073750107 16 0 415 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
a:link, span.MsoHyperlink
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	mso-style-link:"No Spacing Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
span.apple-converted-space
	{mso-style-name:apple-converted-space;
	mso-style-unhide:no;}
p.Code, li.Code, div.Code
	{mso-style-name:Code;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"No Spacing";
	mso-style-link:"Code Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Courier New";
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;}
span.NoSpacingChar
	{mso-style-name:"No Spacing Char";
	mso-style-priority:1;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"No Spacing";
	mso-ansi-font-size:11.0pt;
	mso-bidi-font-size:11.0pt;}
span.CodeChar
	{mso-style-name:"Code Char";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-parent:"No Spacing Char";
	mso-style-link:Code;
	mso-ansi-font-size:11.0pt;
	mso-bidi-font-size:11.0pt;
	font-family:"Courier New";
	mso-ascii-font-family:"Courier New";
	mso-hansi-font-family:"Courier New";
	mso-bidi-font-family:"Courier New";}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-ascii-font-family:Tahoma;
	mso-hansi-font-family:Tahoma;
	mso-bidi-font-family:Tahoma;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-size:10.0pt;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;
	mso-header-margin:.5in;
	mso-footer-margin:.5in;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-para-margin:0in;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1026"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US link=blue vlink=purple style='tab-interval:.5in'>

<div class=WordSection1>

<p class=MsoNoSpacing>February 3, 2012</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Went to the Orientations of Proteins in Membranes
database, went to the &quot;Beta-barrel transmembrane superfamilies&quot; page,
and saved the source code as &quot;opm beta barrel tm.html&quot;</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Downloaded FASTA format sequences for each of the 98
listed beta barrels, using the following code:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>import
re<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>from
urllib import urlretrieve<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>pdbids
= set()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>with
open('opm beta barrel tm.html', 'r') as f:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>for line in f:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>search =
re.search('protein.php\?pdbid=(....)', line)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>if search is not None:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>pdbids.add(search.group(1))<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>for
pdbid in pdbids:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>   
</span>urlretrieve('http://www.rcsb.org/pdb/download/' +\<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>               
</span>'downloadFile.do?fileFormat=FASTA&amp;compression=NO&amp;' +\<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                </span>'structureId=' + pdbid.upper(),<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                </span>'bbtm
sequences/{}.fasta'.format(pdbid))<o:p></o:p></span></p>

<p class=MsoNoSpacing>I renamed the folder they went to &quot;98 bbtm
sequences&quot;, so it will be ovbious when it's out of date.</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>April 6, 2012</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Created the &quot;comparing structure predictions&quot;
folder for comparing alignments created by different methods by their ability
to predict the position of residues in one protein using the structure of
another, homologous protein. </p>

<p class=MsoNoSpacing>- Later, merged with &quot;structure comparisons
folder&quot;, which I had forgotten about</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Downloaded &quot;1A0S_cluster73_BBTM_out.fas&quot;, an
MSA Daniel created, from an e-mail he sent me today. This MSA was created using
one of Jie Liang's BBTM matrices, specifically BBTM OUT. &quot;cluster73&quot;
in the title refers to an HHOMP cluster.</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I searched through &quot;hhomp location of dataset/search
results&quot; for any search result mentioning cluster73 using, in that folder,
the shell command &quot;<span style='font-size:10.0pt;font-family:"Courier New"'>grep
cluster73 ./* &gt; ../../comparing\ structure\ predictions/cluster73\
search.txt</span>&quot;</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>The output was this:</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:<span
style='mso-spacerun:yes'>  </span>1 cluster73<span style='mso-spacerun:yes'>   
</span>--- OMP.18.1.1 La<span style='mso-spacerun:yes'>    </span>100.0<span
style='mso-spacerun:yes'>          </span>0<span style='mso-spacerun:yes'> 
</span>747.4<span style='mso-spacerun:yes'>  </span>40.6 112.4<span
style='mso-spacerun:yes'>  </span>409<span style='mso-spacerun:yes'>   
</span>2-413<span style='mso-spacerun:yes'>    </span>93-511 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:&gt;cluster73<span
style='mso-spacerun:yes'>    </span>--- OMP.18.1.1 LamB-like --- OMP.18.1.2
LamB-like<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:T
cluster73<span style='mso-spacerun:yes'>        </span>93
GVDFHGYARSGILMNDSGGSGEGGPQQCFQAGGAGSKYGRLGNECDTYAELKLGQELYKDNGKSFYFDSMVAYSVKQSND<span
style='mso-spacerun:yes'>  </span>172 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:T
cluster73<span style='mso-spacerun:yes'>       </span>173
WTAEDSDLALRQANVQAKGLPPFLGAFPGATLWAGKRFYQRNFDIHILDFYYWDISGTGAGIEDVDLNGDGKLSFALS-G<span
style='mso-spacerun:yes'>  </span>251 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:T
cluster73<span style='mso-spacerun:yes'>       </span>252
RNDGDNDDQDPNVNRYDLRLAGIPLNPGGSLELGVSYGRANPNDDQKLADGALAGTDAAKDGWMLTAEHTQDSFLGLREG<span
style='mso-spacerun:yes'>  </span>331 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:T
cluster73<span style='mso-spacerun:yes'>       </span>332
FNKFALQYGTDGLGAEVKGIGSGGGTGLDNGAKTWRLIEYGVTPLGDRWELAPALVYQKSKDRYVDGDDYRWLSVGVRPM<span
style='mso-spacerun:yes'>  </span>411 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:T
cluster73<span style='mso-spacerun:yes'>       </span>412
YKWTDNFKTALELGYDYVKLQPTGYKGRQGGTGSLYKFTLAPTWKQAGP--SFWSRPEIRVFATYADWNEALDRAAAGDA<span
style='mso-spacerun:yes'>  </span>489 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1A0S.export:T
cluster73<span style='mso-spacerun:yes'>       </span>490
FGDSGFGDGDEWNFGVQMEAW<span style='mso-spacerun:yes'>  </span>510 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:<span
style='mso-spacerun:yes'>  </span>2 cluster73<span style='mso-spacerun:yes'>   
</span>--- OMP.18.1.1 La<span style='mso-spacerun:yes'>    </span>100.0<span
style='mso-spacerun:yes'>          </span>0<span style='mso-spacerun:yes'> 
</span>765.1<span style='mso-spacerun:yes'>  </span>41.9 108.2<span
style='mso-spacerun:yes'>  </span>385<span style='mso-spacerun:yes'>   
</span>1-421<span style='mso-spacerun:yes'>    </span>94-511 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:&gt;cluster73<span
style='mso-spacerun:yes'>    </span>--- OMP.18.1.1 LamB-like --- OMP.18.1.2
LamB-like<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:T
cluster73<span style='mso-spacerun:yes'>        </span>94
VDFHGYARSGILMNDSGGSGEGGPQQCFQAGGAGSKYGRLGNECDTYAELKLGQELYKDNGKSFYFDSMVAYSVKQSNDW<span
style='mso-spacerun:yes'>  </span>173 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>174
TAEDSDLALRQANVQAKGLPPFLGAFPGATLWAGKRFYQRNFDIHILDFYYWDISGTGAGIEDVDLNGDGKLSFALSGRN<span
style='mso-spacerun:yes'>  </span>253 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>254
DGD------------NDDQDPNVNRYDLRLAGIPLNPGGSLELGVSYGRANPNDDQKLADGALAGTDAAKDGWMLTAEHT<span
style='mso-spacerun:yes'>  </span>321 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>322
QDSFLGLREGFNKFALQYGTDGLGAEVKGIGSGGG---------TGLDNGAKTWRLIEYGVTPLGDRWELAPALVYQKSK<span
style='mso-spacerun:yes'>  </span>392 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>393
DRYVDGDDYRWLSVGVRPMYKWTDNFKTALELGYDYVKLQPTGYKGRQGGTGSLYKFTLAPTWKQAGPSFWSRPEIRVFA<span
style='mso-spacerun:yes'>  </span>472 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./1AF6__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>473
TYADWNEALDRAAAGD---------------AFGDSGFGDGDEWNFGVQMEAW<span
style='mso-spacerun:yes'>  </span>510 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./3PRN__.export:
63 cluster73<span style='mso-spacerun:yes'>    </span>--- OMP.18.1.1 La<span
style='mso-spacerun:yes'>     </span>97.5<span style='mso-spacerun:yes'>   
</span>4.9E+02<span style='mso-spacerun:yes'>   </span>-8.1<span
style='mso-spacerun:yes'>  </span>25.6<span style='mso-spacerun:yes'> 
</span>66.6<span style='mso-spacerun:yes'>  </span>263<span
style='mso-spacerun:yes'>    </span>2-289<span style='mso-spacerun:yes'>   
</span>94-449 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./3PRN__.export:&gt;cluster73<span
style='mso-spacerun:yes'>    </span>--- OMP.18.1.1 LamB-like --- OMP.18.1.2
LamB-like<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./3PRN__.export:T
cluster73<span style='mso-spacerun:yes'>        </span>94
VDFHGYARSGILMNDSGGSGEGGPQQCFQAGGAGSKYGRLGNECDTYAELKLGQELYKDNGKSFYFDSMVAYSVKQSNDW<span
style='mso-spacerun:yes'>  </span>173 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./3PRN__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>174
TAEDSDLALRQANVQAKGLPPFLGAFPGATLWAGKRFYQRNFDIHILDFYYWDISGTGAGIEDV----------------<span
style='mso-spacerun:yes'>  </span>237 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./3PRN__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>238
---------DLNGDGKLSFALSGRNDGDNDDQDPNVNRYDLRLAGIPLNPGGSLELGVSYGRANPNDDQKLADGALAGTD<span
style='mso-spacerun:yes'>  </span>308 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./3PRN__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>309
AAKDGWMLTAEHTQDSFLGLREGFNKFALQYGTDGLGAEVKGIGSGGGTGLDNGAKTWRLIEYGVTPLGDRWELAPALVY<span
style='mso-spacerun:yes'>  </span>388 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>./3PRN__.export:T
cluster73<span style='mso-spacerun:yes'>       </span>389
QKSKDRYVDGDDYRWLSVGVRPMYKWTDNFKTALELGYDYVKLQPTGYKGRQGGTGSLYK<span
style='mso-spacerun:yes'>  </span>448 (512)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing>Those are 1A0S, the sucrose porin ScrY from <i
style='mso-bidi-font-style:normal'>Salmonella typhimurium</i>; 1AF6, the
maltoporin LamB from <i style='mso-bidi-font-style:normal'>Escherichia coli</i>;
and 3PRN, the &quot;general diffusion porin from <i style='mso-bidi-font-style:
normal'>Rhodobacter blastica&quot;, </i>to quote the paper reporting the
structure.</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Counted by hand number of sequences in cluster73, 18.1.1,
and 18.1.2. cluster73 has 51 sequences. Out of 18.1.1 and 18.1.2, one has 13
and the other has 38 sequences. This indicates that cluster73 is simply a
combination of two other clusters.</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>The cluster &quot;OMP.18.1.1 LamB-like&quot;, on the
HHOMP database website at the page <a
href="http://toolkit.tuebingen.mpg.de/hhomp/browse?page=OMP.18.1.1">http://toolkit.tuebingen.mpg.de/hhomp/browse?page=OMP.18.1.1</a>,
has the description:</p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal;background:white'><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:"Times New Roman";
color:black'>This cluster contains proteins annotated as maltoporin, as well as
the PDB structure of the malteporin LamB from Salmonella typhimurium (PDB
identifier&nbsp;<a
href="http://www.pdb.org/pdb/explore/explore.do?structureId=2MPR"><b><span
style='font-size:8.5pt;color:#005FA9;text-decoration:none;text-underline:none'>2MPR</span></b></a>).&nbsp;<br>
<br>
The cluster belongs to a group of clusters with proteins annotated as
LamB-like. The clusters are related and located next to each other in the
cluster map.&nbsp;<br>
<br>
LamB is a outer membrane channel with 18 beta-strands specific for the transport
of maltose and maltodextrines. The specifity of this channel is conferred by
outer loops which fold back into the barrel. This protein forms a homo-trimer
composed of three identical subunits.&nbsp;<br>
<br>
Most proteins in this cluster are from gamma-proteobacteria.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
mso-fareast-font-family:"Times New Roman"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
0%;background:white'><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:"Times New Roman";
color:black'>&nbsp;<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal;background:white'><b><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:"Times New Roman";
color:black'>length of HMM:&nbsp;</span></b><span style='font-size:7.5pt;
font-family:"Verdana","sans-serif";mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black'>406
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>approximate
position of beta-barrel domain:&nbsp;</b>26 - 406&nbsp;<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>And &quot;OMP.18.1.2 LamB-like&quot;, at <a
href="http://toolkit.tuebingen.mpg.de/hhomp/browse?page=OMP.18.1.2">http://toolkit.tuebingen.mpg.de/hhomp/browse?page=OMP.18.1.2</a>,
has the description:</p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal;background:white'><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:"Times New Roman";
color:black'>This cluster contains proteins annotated as sucrose porin.&nbsp;<br>
<br>
The cluster belongs to a group of clusters with proteins annotated as
LamB-like. The clusters are related and located next to each other in the
cluster map. One of these clusters (<a
href="http://toolkit.tuebingen.mpg.de/hhomp/browse?page=OMP.18.1.1"><b><span
style='font-size:8.5pt;color:#005FA9;text-decoration:none;text-underline:none'>OMP.18.1.1</span></b></a>)
consists the PDB structure of the malteporin LamB from Salmonella typhimurium
(PDB identifier&nbsp;<a
href="http://www.pdb.org/pdb/explore/explore.do?structureId=2MPR"><b><span
style='font-size:8.5pt;color:#005FA9;text-decoration:none;text-underline:none'>2MPR</span></b></a>).&nbsp;<br>
<br>
LamB is a outer membrane channel with 18 beta-strands specific for the
transport of maltose and maltodextrines. The specifity of this channel is
conferred by outer loops which fold back into the barrel. This protein forms a
homo-trimer composed of three identical subunits.&nbsp;<br>
<br>
All proteins in this cluster are from gamma-proteobacteria.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
mso-fareast-font-family:"Times New Roman"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
0%;background:white'><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:"Times New Roman";
color:black'>&nbsp;<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal;background:white'><b><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
mso-fareast-font-family:"Times New Roman";mso-bidi-font-family:"Times New Roman";
color:black'>length of HMM:&nbsp;</span></b><span style='font-size:7.5pt;
font-family:"Verdana","sans-serif";mso-fareast-font-family:"Times New Roman";
mso-bidi-font-family:"Times New Roman";color:black'>503
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>approximate
position of beta-barrel domain:&nbsp;</b>92 - 503&nbsp;<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>The cluster73 page, at <a
href="http://toolkit.tuebingen.mpg.de/hhomp/browse?page=cluster73">http://toolkit.tuebingen.mpg.de/hhomp/browse?page=cluster73</a>,
just has </p>

<p class=MsoNoSpacing><strong><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
color:black;background:white'>length of HMM:</span></strong><span
class=apple-converted-space><b><span style='font-size:7.5pt;font-family:"Verdana","sans-serif";
color:black;background:white'>&nbsp;</span></b></span><span style='font-size:
7.5pt;font-family:"Verdana","sans-serif";color:black;background:white'><span
style='orphans: 2;text-align:-webkit-auto;widows: 2;-webkit-text-size-adjust: auto;
-webkit-text-stroke-width: 0px;float:none;word-spacing:0px'>512
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span
class=apple-converted-space>&nbsp;</span></span><strong style='orphans: 2;
text-align:-webkit-auto;widows: 2;-webkit-text-size-adjust: auto;-webkit-text-stroke-width: 0px;
word-spacing:0px'><span style='font-family:"Verdana","sans-serif";mso-bidi-font-family:
"Times New Roman";mso-bidi-theme-font:minor-bidi'>approximate position of
beta-barrel domain:</span></strong><span class=apple-converted-space><b>&nbsp;</b>92
- 511&nbsp;<o:p></o:p></span></span></p>

<p class=MsoNoSpacing><span class=apple-converted-space><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I loaded into ClustalX the sequences of 1A0S and 1AF6
from &quot;structure comparisons/98 bbtm sequences&quot;. These are the FASTA
files from the PDB. Daniel has told me these don't always exactly match the
sequences in the structure.</p>

<p class=MsoNoSpacing>I made an alignment using ClustalX's &quot;Identity
Matrix&quot; option. I wasn't sure whether to set it in the Multiple Alignment
Parameters dialog or the Pairwise Alignment Parameters, so I did both. I saved
the alignment as &quot;structure comparisons/1a0s 1af6 pairwise id
align.clu&quot;.</p>

<p class=MsoNoSpacing>I downloaded &quot;BBTM_out.txt&quot; from an email Daniel
sent me today. He tells me this is a Jie Liang matrix and that this is what he
used to generate alignment of 1A0S with cluster73 that he sent me. The file is:</p>

<p class=MsoNoSpacing># bbTMout</p>

<p class=MsoNoSpacing># Lowest score = -16 , Highest score = 25</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>A
R N D C Q E G H I L K M F P S T W Y V B Z X *<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>A
6 -12 -11 -13 -12 -11 -10 1 -13 -1 -1 -13 -5 -3 -8 -3 0 -9 -8 0 -12 -4 -6 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>R
-12 14 -7 -9 -12 -5 -8 -10 -6 -12 -12 -4 -13 -13 -14 -11 -8 -12 -8 -12 -8 -9 -8
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>N
-11 -7 13 -1 -12 -3 -8 -9 -7 -13 -12 -7 -8 -12 -14 -3 -3 -14 -10 -12 6 -8 -7 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>D
-13 -9 -1 14 -12 -7 -5 -8 -12 -13 -11 -6 -12 -13 -14 -8 -11 -14 -11 -13 7 -6 -8
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>C
-14 -13 -11 -12 25 -11 -11 -14 -14 -14 -15 -14 -14 -14 -13 -14 -12 -14 -11 -13
-11 -12 -11 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>Q
-11 -5 -3 -7 -10 12 -4 -12 -2 -12 -12 -6 -9 -12 -10 -8 -4 -10 -8 -11 -5 -8 -7
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>E
-10 -8 -8 -5 -12 -4 17 -11 -12 -12 -12 -13 -14 -12 -16 -11 -8 -13 -13 -11 -6 3
-9 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>G
1 -10 -9 -8 -12 -12 -11 8 -14 -6 -6 -13 -9 -7 -9 -4 -3 -11 -10 -5 -8 -1 -7 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>H
-13 -6 -7 -12 -12 -2 -12 -14 13 -12 -12 -10 -13 -11 -14 -9 -12 -9 -5 -13 -9 -13
-9 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>I
-1 -12 -13 -13 -13 -12 -12 -6 -12 5 2 -14 -3 -1 -13 -9 -5 -6 -8 3 -13 -9 -7 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>L
-1 -12 -12 -11 -15 -12 -12 -6 -12 2 4 -13 -2 1 -11 -9 -5 -5 -6 2 -11 -9 -6 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>K
-13 -4 -7 -6 -12 -6 -13 -13 -10 -14 -13 14 -12 -12 -12 -10 -9 -13 -12 -13 -6
-13 -9 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>M
-5 -13 -8 -12 -12 -9 -14 -9 -13 -3 -2 -12 12 -3 -13 -10 -4 -9 -9 -3 -10 -11 -7
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>F
-3 -13 -12 -13 -12 -12 -12 -7 -11 -1 1 -12 -3 6 -10 -10 -8 -3 -2 0 -12 -9 -6
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>P
-8 -14 -14 -14 -12 -10 -16 -9 -14 -13 -11 -12 -14 -10 12 -8 -9 -13 -15 -12 -14
-12 -10 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>S
-3 -11 -3 -8 -12 -8 -11 -4 -9 -9 -9 -10 -10 -10 -8 11 -1 -12 -11 -8 -5 -7 -7
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>T
0 -8 -3 -11 -12 -4 -8 -3 -12 -5 -5 -9 -4 -8 -9 -1 9 -11 -8 -3 -7 -5 -5 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>W
-9 -12 -14 -14 -12 -10 -13 -11 -9 -6 -5 -13 -9 -3 -13 -12 -11 9 -1 -5 -14 -12
-8 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>Y
-8 -8 -10 -11 -12 -8 -13 -10 -5 -8 -6 -12 -9 -2 -15 -11 -8 -1 6 -7 -10 -11 -7
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>V
0 -12 -12 -13 -13 -11 -11 -5 -13 3 2 -13 -3 0 -12 -8 -3 -5 -7 4 -12 -8 -6 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>B
-12 -8 6 7 -11 -5 -6 -8 -9 -13 -11 -6 -10 -12 -14 -5 -7 -14 -10 -12 14 -5 -8
-16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>Z
-4 -9 -8 -6 -12 -8 3 -1 -13 -9 -9 -13 -11 -9 -12 -7 -5 -12 -11 -8 -5 15 -8 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>X
-6 -8 -7 -8 -11 -7 -9 -7 -9 -7 -6 -9 -7 -6 -10 -7 -5 -8 -7 -6 -8 -8 1 -16<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>*
-16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16 -16
-16 -16 -16 1<o:p></o:p></span></p>

<p class=MsoNoSpacing>I did the alignment again, this time selecting
&quot;Reset All Gaps before Alignment&quot;, and in the Parwise Alignment
Parameters and Multiple Alignment Parameters dialogs selecting User Selected
and loading &quot;structure comparisons/BBTM_out.txt&quot;, the file whose
contents are above. I saved the alignment as &quot;structure comparisons/1a0s
1af6 pairwise bbtmout align.clu&quot;</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I compared sequence identity in these two alignments,
using the Python interpreter interactively.. Here is an abridged version of the
python session:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>Python
2.7.2 (default, Jun 12 2011, 15:08:59) [MSC v.1500 32 bit (Intel)] on win32<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>Type
&quot;copyright&quot;, &quot;credits&quot; or &quot;license()&quot; for more
information.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
import os<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
os.chdir(r'C:\cygwin\home\alex\beta barrels\structure comparisons')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
from __future__ import division<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
from Bio import AlignIO<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
id = AlignIO.read('1a0s 1af6 pairwise id align.clu', 'clustal')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
bbtm = AlignIO.read('1a0s 1af6 pairwise bbtmout align.clu', 'clustal')<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>April 13, 2012</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Going through the steps from last time's Python session.</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>Python
2.7.2 (default, Jun 12 2011, 15:08:59) [MSC v.1500 32 bit (Intel)] on win32<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>Type
&quot;copyright&quot;, &quot;credits&quot; or &quot;license()&quot; for more
information.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
import os<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
os.chdir(r'C:\cygwin\home\alex\beta barrels\structure comparisons')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
from __future__ import division<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
from Bio import AlignIO<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
id = AlignIO.read('1a0s 1af6 pairwise id align.clu', 'clustal')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
bbtm = AlignIO.read('1a0s 1af6 pairwise bbtmout align.clu', 'clustal')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
idcount = 0<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
for index, resn in enumerate(id[0].seq):<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-tab-count:1'>      </span>if id[1].seq[index] == resn:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-tab-count:2'>            </span>idcount += 1<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-tab-count:2'>            </span><o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
idcount/len(id[0].seq)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>0.2660753880266075<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
bbtmcount = 0<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
for index, resn in enumerate(bbtm[0].seq):<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-tab-count:1'>      </span>if bbtm[1].seq[index] == resn:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-tab-count:2'>            </span>bbtmcount += 1<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-tab-count:2'>            </span><o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
bbtmcount/len(bbtm[0].seq)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>0.14018691588785046<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>So when aligned with BBTM, there is 14% sequence identity
between the two.</p>

<p class=MsoNoSpacing>More interesting would be to know the BBTM score between
the two. That way, if we have a multiple sequence alignment that we aligned
with BBTM, we can order it by BBTM score. And we'll know that, whatever the structure
prediction results are for this pair, those are indicative of what the results
would be for the pair on the list with around the same BBTM score. But I'll
figure that out once I've got the comparison.</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I want to check whether the FASTAS I downloaded from the
PDB are the same as the sequences of Daniel's aligned structures, for 1A0S and
1AF6. I know in at least one case they aren't: 1FEP has no methionines in
Daniel's aligned structure, by some mistake, but I expect that the methionine's
are present in the FASTA sequence. He has also told me that they are not always
the same, so he must have observed differences somewhere.</p>

<p class=MsoNoSpacing>Abridged PyMOL session where I retrieve the sequences
from Daniel's aligned structures:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;stored.scryseq
= ''<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;print(one_letter)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>{'CYS':
'C', 'ILE': 'I', 'GLN': 'Q', 'VAL': 'V', 'LYS': 'K', 'THR': 'T', 'PRO': 'P',
'GLY': 'G', 'HIS': 'H', 'PHE': 'F', 'ASN': 'N', 'MET': 'M', 'ASP': 'D', 'LEU':
'L', 'ARG': 'R', 'TRP': 'W', 'HOH': '', 'ALA': 'A', 'GLU': 'E', 'TYR': 'Y',
'SER': 'S'}<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;iterate
1a0s.molecule &amp; n. ca, stored.scryseq += one_letter[resn]<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'> </span>Iterate: iterated over 413 atoms.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;print(stored.scryseq)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>SGFEFHGYARSGVIMNDSGASTKSGAYITPAGETGGAIGRLGNQADTYVEMNLEHKQTLDNGATTRFKVMVADGQTSYNDWTASTSDLNVRQAFVELGNLPTFAGPFKGSTLWAGKRFDRDNFDIHWIDSDVVFLAGTGGGIYDVKWNDGLRSNFSLYGRNFGDIDDSSNSVQNYILTMNHFAGPLQMMVSGLRAKDNDERKDSNGNLAKGDAANTGVHALLGLHNDSFYGLRDGSSKTALLYGHGLGAEVKGIGSDGALRPGADTWRIASYGTTPLSENWSVAPAMLAQRSKDRYADGDSYQWATFNLRLIQAINQNFALAYEGSYQYMDLKPEGYNDRQAVNGSFYKLTFAPTFKVGSIGDFFSRPEIRFYTSWMDWSKKLNNYASDDALGSDGFNSGGEWSFGVQMETWF<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;stored.maltoporinseq
= ''<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;iterate
1af6.molecule &amp; n. ca, stored.maltoporinseq += one_letter[resn]<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'> </span>Iterate: iterated over 421 atoms.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;print(stored.maltoporinseq)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>VDFHGYARSGIGWTGSGGEQQCFQTTGAQSKYRLGNECETYAELKLGQEVWKEGDKSFYFDTNVAYSVAQQNDWEATDPAFREANVQGKNLIEWLPGSTIWAGKRFYQRHDVHMIDFYYWDISGPGAGLENIDVGFGKLSLAATRSSEAGGSSSFASNNIYDYTNETANDVFDVRLAQMEINPGGTLELGVDYGRANLRDNYRLVDGASKDGWLFTAEHTQSVLKGFNKFVVQYATDSMTSQGKGLSQGSGVAFDNEKFAYNINNNGHMLRILDHGAISMGDNWDMMYVGMYQDINWDNDNGTKWWTVGIRPMYKWTPIMSTVMEIGYDNVESQRTGDKNNQYKITLAQQWQAGDSIWSRPAIRVFATYAKWDEKWGYDYTGNADNNANFGKAVPADFNGGSFGRGDSDEWTFGAQMEIWW<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Having retrieved these sequences, I returned to the
Python session from above, which was still open, and added variables containing
these sequences from PyMOL using copy and paste:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
scryseq_from_pymol =
'SGFEFHGYARSGVIMNDSGASTKSGAYITPAGETGGAIGRLGNQADTYVEMNLEHKQTLDNGATTRFKVMVADGQTSYNDWTASTSDLNVRQAFVELGNLPTFAGPFKGSTLWAGKRFDRDNFDIHWIDSDVVFLAGTGGGIYDVKWNDGLRSNFSLYGRNFGDIDDSSNSVQNYILTMNHFAGPLQMMVSGLRAKDNDERKDSNGNLAKGDAANTGVHALLGLHNDSFYGLRDGSSKTALLYGHGLGAEVKGIGSDGALRPGADTWRIASYGTTPLSENWSVAPAMLAQRSKDRYADGDSYQWATFNLRLIQAINQNFALAYEGSYQYMDLKPEGYNDRQAVNGSFYKLTFAPTFKVGSIGDFFSRPEIRFYTSWMDWSKKLNNYASDDALGSDGFNSGGEWSFGVQMETWF'<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
maltoporinseq_from_pymol = 'VDFHGYARSGIGWTGSGGEQQCFQTTGAQSKYRLGNECETYAELKLGQEVWKEGDKSFYFDTNVAYSVAQQNDWEATDPAFREANVQGKNLIEWLPGSTIWAGKRFYQRHDVHMIDFYYWDISGPGAGLENIDVGFGKLSLAATRSSEAGGSSSFASNNIYDYTNETANDVFDVRLAQMEINPGGTLELGVDYGRANLRDNYRLVDGASKDGWLFTAEHTQSVLKGFNKFVVQYATDSMTSQGKGLSQGSGVAFDNEKFAYNINNNGHMLRILDHGAISMGDNWDMMYVGMYQDINWDNDNGTKWWTVGIRPMYKWTPIMSTVMEIGYDNVESQRTGDKNNQYKITLAQQWQAGDSIWSRPAIRVFATYAKWDEKWGYDYTGNADNNANFGKAVPADFNGGSFGRGDSDEWTFGAQMEIWW'<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I loaded into Python variables the FASTA sequences used
in the multiple sequence alignments, from the folder '98 bbtm sequences':</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
from Bio import SeqIO<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
scryseq_from_fasta = SeqIO.parse('98 bbtm sequences/1a0s.fasta', 'fasta')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
maltoporinseq_from_fasta = SeqIO.parse('98 bbtm sequences/1af6.fasta', 'fasta')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing>I verified in ClustalX that the alignments I made used
chain P from ScrY and chain A from maltoporin. I verified in PyMOL that
Daniel's structures are also those chains.</p>

<p class=MsoNoSpacing><br>
SeqIO.parse returns generators, so I converted them into lists:</p>

<p class=Code>&gt;&gt;&gt; scryseq_from_fasta = list(scryseq_from_fasta)<o:p></o:p></p>

<p class=Code>&gt;&gt;&gt; maltoporinseq_from_fasta =
list(maltoporinseq_from_fasta)</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I found that the first entries of these lists are chain P
and chain A, respectively:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
scryseq_from_fasta[0]<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>SeqRecord(seq=Seq('SGFEFHGYARSGVIMNDSGASTKSGAYITPAGETGGAIGRLGNQADTYVEMNLE...TWF',
SingleLetterAlphabet()), id=<span style='color:#C00000'>'<b style='mso-bidi-font-weight:
normal'>1A0S:P|PDBID|CHAIN|SEQUENCE'</b></span>,
name='1A0S:P|PDBID|CHAIN|SEQUENCE', description='1A0S:P|PDBID|CHAIN|SEQUENCE',
dbxrefs=[])<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>&gt;&gt;&gt;
maltoporinseq_from_fasta[0]<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>SeqRecord(seq=Seq('VDFHGYARSGIGWTGSGGEQQCFQTTGAQSKYRLGNECETYAELKLGQEVWKEG...IWW',
SingleLetterAlphabet()), id=<span style='color:#C00000'>'<b style='mso-bidi-font-weight:
normal'>1AF6:A|PDBID|CHAIN|SEQUENCE'</b></span>,
name='1AF6:A|PDBID|CHAIN|SEQUENCE', description='1AF6:A|PDBID|CHAIN|SEQUENCE',
dbxrefs=[])<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing>I checked to see whether these were the same as the
sequences from PyMOL and it turns out that, at least in this case, the FASTA
files and the sequences of Daniel's aligned structures are identical.</p>

<p class=Code>&gt;&gt;&gt; str(scryseq_from_fasta[0].seq) == scryseq_from_pymol<o:p></o:p></p>

<p class=Code>True<o:p></o:p></p>

<p class=Code>&gt;&gt;&gt; str(maltoporinseq_from_fasta[0].seq) ==
maltoporinseq_from_pymol<o:p></o:p></p>

<p class=Code>True</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Daniel just told me that 1E54's FASTA has an opening
&quot;E&quot;, but the PDB file actually begins with a heteroatom, and there's
no glutamate at the beginning of it.</p>

<p class=MsoNoSpacing>I downloaded a file, &quot;FASTAs_New.rar&quot;, that
Daniel sent me today, which has FASTA files which are sequences extracted from
the aligned structures. This means that at least 1FEP has missing methionines.
I saved these to the folder &quot;structure comparisons/daniel's aligned
structure sequences'.</p>

<p class=MsoNoSpacing>I fetched pdb structures corresponding to all of Daniel's
aligned structures:</p>

<p class=Code>PyMOL&gt;for name in groupdict.keys(): cmd.fetch(name, name +
'frompdb')</p>

<p class=MsoNoSpacing>And then checked which one of these had
selenomethionines, as indicated by the residue name &quot;MSE&quot;. I found
that 2VQI also had these selenomethionine residues, and verified visually that,
as in 1FEP, they were missing from Daniel's aligned structure. Daniel says that
he used a sript that removed all heteroatoms which did so by examining residue
name. The script must have removed selenomethionines. The following is the code
I used to search for &quot;MSE&quot;:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>stored.has_mse
= set()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>for
name in groupdict.keys():<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>stored.name = name<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>cmd.iterate(name+'frompdb &amp; resn mse',
'stored.has_mse.add(stored.name)')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>PyMOL&gt;print
stored.has_mse<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>set(['1FEP',
'2VQI'])<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I examined Daniel's aligned structures of 1AF6 and 1A0S
and took note of the locations of the strands, and highlighted the strands in a
text representation of the BBTM_OUT alignment, producing this:</p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>First line: 1A0S<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>Second line: 1AF6<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>  </span>1<span style='mso-tab-count:9'>                                        </span><span
style='mso-spacerun:yes'>   </span>2<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>SG<span style='color:#C00000'>FEFHGYARSGVIM</span>NDSGASTKSGAYITPAGETGGAIGRLGNQAD<span
style='color:#C00000'>TYVE</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>-------V<span
style='color:#C00000'>DFHGYARSGIGWT</span>GSGGEQQCFQTTGAQSKYRLGNECE<span
style='color:#C00000'>TYAE</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span style='mso-tab-count:
1'>     </span><span style='mso-spacerun:yes'>   </span>1<span
style='mso-tab-count:8'>                                  </span><span
style='mso-spacerun:yes'>   </span>2<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span>3<span
style='mso-spacerun:yes'>                          </span>4<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>MNLEHKQT</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>LDN<span style='color:#C00000'>GATTRFKVMVADG</span>QTSYNDWTASTSDL<span
style='color:#C00000'>NVRQAFVEL</span>GNL<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>LKLGQEVWKE</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>GD<span style='color:#C00000'>KSFYFDTNVAYSV</span>AQQNDWEATDP<span
style='color:#C00000'>AFREANVQG</span>KNLIE<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>3<span
style='mso-spacerun:yes'>                       </span>4<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span><span
style='mso-spacerun:yes'>      </span>5<span
style='mso-spacerun:yes'>                        </span>6<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>PTFAGPFKG--------S<span
style='color:#C00000'>TLWAGKRF</span>DRDNFDIHWIDSDVVFL<span style='color:#C00000'>AGTGGGI</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>WLPGS<span style='color:
#C00000'>TIWA</span>GKRFYQRHDVHMIDFYYWDISG<span style='color:#C00000'>PGAGL</span>ENIDVGFGK<span
style='color:#C00000'>LSLAA</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>     </span>5<span
style='mso-spacerun:yes'>                         </span>6<span
style='mso-spacerun:yes'>            </span><span
style='mso-spacerun:yes'> </span>7<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>       </span>7<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>         </span>8<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>   </span>9<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>YDVKW</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>ND<span style='color:#C00000'>GLRSNFSLYGRNF</span>GDIDDSSN<span
style='color:#C00000'>SVQNYILTMNHFA</span>GP<span style='color:#C00000'>LQMMVSG</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>TRSSE</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>AGGSSSFASNNIYDYTNE<span style='color:#C00000'>TANDVFDVRLAQME</span>INPG<span
style='color:#C00000'>GTLELGVDY</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                       </span>8<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>      </span>9<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'> </span>10<span style='mso-spacerun:yes'>          
</span><span style='mso-spacerun:yes'>       </span>11<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>LR</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>AKDNDERKDSNGNLAKGDAAN<span style='color:#C00000'>TGVHALLGLHN</span>DSFYGLRDG<span
style='color:#C00000'>SSKTALL</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>GRA</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>NLRDNYRLVDGASK<span style='color:#C00000'>DGWLFTAEHTQS</span>VLKG<span
style='color:#C00000'>FNKFVVQYAT</span>DSMTSQG<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>      </span>10<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>   </span>11<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>           </span>12<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>   </span>13<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>YGH</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>GLGAEVKGIGSDGALRPGA<span style='color:#C00000'>DTWRIASYGTTP</span>LSEN<span
style='color:#C00000'>WSVAPAMLAQRS</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>KGLSQGSGVAFDNEKFAYNINNN<span
style='color:#C00000'>GHMLRILDHGAISM</span>GDN<span style='color:#C00000'>WDMMYVGMYQ</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'> </span>12<span style='mso-spacerun:yes'>          
</span><span style='mso-spacerun:yes'>    </span>13<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>14<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>     </span>15<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>K</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>DRYADGD<span style='color:#C00000'>SYQWATFNLRLIQ</span>AINQN<span
style='color:#C00000'>FALAYEGSYQYMDLK</span>PEGYNDRQA<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>DINW</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>DNDNG<span style='color:#C00000'>TKWWTVGIRPMYK</span>WTPI<span
style='color:#C00000'>MSTVMEIGYDNVE</span>SQRT<span style='color:#C00000'>GDKNNQY</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>         </span>14<span style='mso-spacerun:yes'>   
</span><span style='mso-spacerun:yes'>           </span>15<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>    </span>16<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>16<span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>  </span>17<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>VNGSFYKLTFAPTF</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>KVGSIGDFFSRP<span style='color:#C00000'>EIRFYTSWMDW</span>SKKLNN-------<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'>KITLAQQW</span></b><b
style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;font-family:
"Courier New"'>QAGDSIWSRP<span style='color:#C00000'>AIRVFATYAKWD</span>EKWGYDYTGNADNNANFGKA<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>       </span>17<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>     </span>18<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>YASDDALGSDGFNSGG<span
style='color:#C00000'>EWSFGVQMETWF</span><o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>VPADFNGGSFGRG<span
style='color:#C00000'>DSDEWTFGAQMEIWW<o:p></o:p></span></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>           </span><span
style='mso-spacerun:yes'>  </span>18<span style='color:#C00000'><o:p></o:p></span></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New";color:#C00000'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>18 strands in 1A0S<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;font-family:"Courier New"'>18 strands in 1AF6<o:p></o:p></span></b></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Strands 2, 3, 4, 9, 12, 13, 14, 15, 16 and 18 pretty much
line up.</p>

<p class=MsoNoSpacing>Looking at the structure of 1Af6 in PyMOL, I see that
loop 7 of 1A0S is aligned with an extracellular loop in 1AF6. If 1A0S were a
protein of known structure and the structure corresponding to 1AF6's sequence
was unknown, and we used this alignment combined with the structure of 1A0S to
estimate the z coordinates in 1AF6's sequence, we would give that loop
transmembrane z values.</p>

<p class=MsoNoSpacing>Structurally, 1AF6 and 1A0S align very, very closely. See
the following images, made using Daniel's aligned structures of 1A0S and 1AF6
and PyMOL's &quot;align&quot; command:</p>

<p class=MsoNoSpacing><span style='mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_1" o:spid="_x0000_i1025" type="#_x0000_t75"
 alt="Description: C:\Users\Nanda Lab\Documents\1af6_1a0s.png" style='width:480pt;
 height:306pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="comparing%20structure%20predictions%20log_files/image001.png"
  o:title="1af6_1a0s"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=640 height=408
src="comparing%20structure%20predictions%20log_files/image001.png"
alt="Description: C:\Users\Nanda Lab\Documents\1af6_1a0s.png" v:shapes="Picture_x0020_1"><![endif]></span></p>

<p class=MsoNoSpacing>Differences in strand lengths observed in the annotated
sequences above don't reflect real divergence in position, but rather just
enough change for PyMOL's secondary structure prediction to give different
assignments. It would be possible, then, to make by hand an alignment in which
the beta strandsline up perfectly.</p>

<p class=MsoNoSpacing>The residues in 1A0S that are aligned with 1AF6's TM
strand 10 are half in the TM region, and half in a loop that goes high above
the protein.</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>I wrote code to calculate the divergence in z coordinates
in the hypothetical situation that the structure of 1A0S was used to predict
the z-values of 1AF6's sequence and vice versa, using the BBTM alignments. The
code corresponds to the following procedure:<br>
Suppose you are using 1A0S to predict the z-values of a protein with 1AF6's
sequence.</p>

<p class=MsoNoSpacing>Make a list of the z coordinates of each C-&#945; in
1A0S, from low resi to high, writing each on small stones and placing them in
order. Do the same for 1AF6.</p>

<p class=MsoNoSpacing>Place the pairwise sequence alignment in front of you.</p>

<p class=MsoNoSpacing>For each position in the sequence alignment, starting
from the left:</p>

<p class=MsoNoSpacing>-if the known structure sequence is a gap, then there's
no position in 1A0S corresponding to that position in 1AF6. Throw away the
top-most z-value stone from the 1AF6 list; there will be no 1A0S z-value
corresponding to it.</p>

<p class=MsoNoSpacing>-if the unknown structure sequence is a gap, then there's
no residue in 1AF6 corresponding to that position in 1A0S. Throw away the
top-most z-value stone from the 1A0S list; there will be no 1AF6 z-value
corresponding to it.</p>

<p class=MsoNoSpacing>-otherwise, subtrac the number on the top 1AF6 stone
(your &quot;real&quot; stone) from the number on the top 1A0S stone (your
&quot;prediction&quot; stone). Write the difference on a piece of paper labeled
&quot;z_diffs&quot;.</p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>This was the code, which should return an output that is
the average of the &quot;z_diff&quot; list:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>from
__future__ import division<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>from
Bio import AlignIO<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>from
sundries import CIDict<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>from
sundries import one_letter<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>from
Bio.PDB import PDBParser<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>import
warnings<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>#
Retrieve the sequences from the BBTMOUT alignment, including -'s for gaps<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>bbtm_align
= list(AlignIO.read('1a0s 1af6 pairwise bbtmout align.clu',<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                          </span>'clustal'))<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>#
Assuming the first is 1A0S, the second is 1AF6:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>sequences
= CIDict((('1A0S',str(bbtm_align[0].seq)),<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                    </span>('1AF6',str(bbtm_align[1].seq))))<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>#
Check that I'm right about the first being 1A0S, the second being 1AF6<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>assert
sequences['1A0S'][:8] == 'SGFEFHGY' \<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>       </span>and sequences['1AF6'][7:11] == 'VDFH',\<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>       </span>'wrong aligned sequences in
&quot;sequences&quot; dictionary'<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>#
Retrieve Daniel's aligned structures<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>structures
= CIDict()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>parser
= PDBParser()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>with
warnings.catch_warnings():<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span># When importing Daniel's aligned
structures, the PDBParser gives<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span># warnings about &quot;invalid or
missing&quot; b factors and occupancies<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span># There are so many that if you let it
display warnings it'll never<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span># finish parsing<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>warnings.simplefilter('ignore')<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>for pdbid in ('1A0S', '1AF6'):<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>structures.update({pdbid: \<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                          
</span>parser.get_structure(pdbid,<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                         </span><span
style='mso-spacerun:yes'>  </span>'aligned_{}.pdb'.format(pdbid))})<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>z_coords
= CIDict()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>for
pdbid in structures.keys():<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>z_coords.update({pdbid: list()})<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>iter_sequence = iter(sequences[pdbid])<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>for residue in
structures[pdbid].get_residues():<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>try:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>calpha = residue.child_dict['CA']<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>except KeyError:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># HOH and other heteroatoms will
not have a C-alpha<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># They will also not have a
corresponding letter in the sequence<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># So, skip them<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>continue<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>resi = residue.get_id()[1]<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>z = calpha.get_coord()[2]<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span># This structure should have the same
sequence as the one<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span># loaded from the multiple sequence
alignment.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span># Raise an error if this residue
doesn't match the next letter<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span># in the sequence alignment:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>seq_equiv = iter_sequence.next()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>while seq_equiv == '-':<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># Skip gaps:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>seq_equiv = iter_sequence.next()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>assert
one_letter[residue.get_resname()] == seq_equiv,\<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>               </span>'Residue with resi {} in pdb
file'.format(resi) \<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>               </span>+ 'doesn\'t match sequence in
{}'.format(pdbid)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span><o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span># Save z coordinate<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>z_coords[pdbid].append(z)<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>z_diff_when_known_is
= CIDict()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>for
known_structure in ('1A0S', '1AF6'):<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>if known_structure == '1AF6':<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>unknown_structure = '1A0S'<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>if known_structure == '1A0S':<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>unknown_structure = '1AF6'<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span><o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>z_diff = list()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>iter_known_z =
iter(z_coords[known_structure])<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>iter_unknown_z =
iter(z_coords[unknown_structure])<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span>for known_seq_letter, unknown_seq_letter \<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>in zip(sequences[known_structure],
sequences[unknown_structure]):<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>if known_seq_letter == '-':<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># There is no position in the known
structure corresponding to<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># this residue in the sequence of
the protein of unknown structure.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># Ignore that residue.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>iter_unknown_z.next()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>if unknown_seq_letter == '-':<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># A position in the known structure
corresponds to a<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># gap. Ignore that position.<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>iter_known_z.next()<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>        </span>else:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># Predicted position minus real
position corresponding to<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span># this sequence position<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>try:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                </span>z_diff.append(iter_known_z.next()
- iter_unknown_z.next())<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>            </span>except StopIteration:<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>                </span>continue<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>    </span><o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><span
style='mso-spacerun:yes'>   
</span>z_diff_when_known_is.update({known_structure: z_diff})<o:p></o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>print(sum([abs(x)
for x in z_diff_when_known_is['1A0S']])/len(z_diff_when_known_is['1A0S']))<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>And the output was:</p>

<p class=MsoNoSpacing><span style='font-size:10.0pt;font-family:"Courier New"'>7.79974133642<o:p></o:p></span></p>

<p class=MsoNoSpacing><o:p>&nbsp;</o:p></p>

<p class=MsoNoSpacing>Which means that this is really not viable as a
prediction method for proteins with this little sequence similarity. However,
there are checks on the accuracy of this code's calculation that I have not yet
done, so I am not confident of this result.</p>

</div>

</body>

</html>
